pom.xml

```diff
@@ -310,6 +310,8 @@ limitations under the License.
                         <exclude>**/test-data/**</exclude>
                         <exclude>**/test-license/**</exclude>
                         <exclude>**/testcases/**</exclude>
+                        <exclude>**/resources/**/*.sql</exclude>
```

Comment:
Is there a reason you are excluding sql files in only specific locations rather than everywhere?

Reply:
@michael-bryson, Now I have updated to exclude all SQL files irrespective of the specific location

Reply:
The settings service build passed after this change.

Reply:
@thilagasanthosh I was not telling you to make this change, I was asking was there a reason to only exclude in specific locations? I don't know if excluding everywhere is valid or not, thats why I was asking you. Should we only be excluding sql files that are executed, what about sql files that are not executed but could be in a repository?

Reply:
@michael-bryson ,
when we discussed in one standup with @dermot-hardy, he told to exclude only with sql files that causing issues.